rust/ip-packet/src/lib.rs

```diff
@@ -172,51 +157,64 @@ impl<'a> ConvertibleIpv4Packet<'a> {
         self.ip_header().destination_addr()
     }
 
-    fn consume_to_ipv6(
-        mut self,
-        src: Ipv6Addr,
-        dst: Ipv6Addr,
-    ) -> Option<ConvertibleIpv6Packet<'a>> {
-        let offset = nat46::translate_in_place(&mut self.buf, src, dst)
-            .inspect_err(|e| tracing::trace!("NAT64 failed: {e:#}"))
-            .ok()?;
-        let buf = self.buf.remove_from_head(offset);
+    fn consume_to_ipv6(mut self, src: Ipv6Addr, dst: Ipv6Addr) -> Option<ConvertibleIpv6Packet> {
+        dbg!(self.start);
```

Comment:
leftover

Reply:
there are a few more below

Reply:
Yep, the last commit was just WIP where I stopped debugging on Friday evening :sweat_smile: